redbot.py

```
@@ -133,24 +135,49 @@ def ensure_logs_directory(self):
         os.makedirs('logs', exist_ok=True)
     
     def osint_google_dorking(self, query: str) -> List[str]:
-        """Realiza Google Dorking para OSINT"""
+        """Realiza OSINT usando maigret para usernames ou Google Dorking"""
+        # Verifica se parece um username (sem espaços, sem operadores especiais)
+        if ' ' not in query and not any(op in query for op in ['site:', 'filetype:', 'inurl:', 'intitle:']):
+            # Tenta usar maigret para username
+            try:
+                result = subprocess.run(
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)